src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1333,31 +1331,25 @@ def load_der_parameters(self, data):
 
         self._handle_key_loading_error()
 
-    def load_pem_x509_certificate(self, data: bytes) -> _Certificate:
-        mem_bio = self._bytes_to_bio(data)
-        x509 = self._lib.PEM_read_bio_X509(
-            mem_bio.bio, self._ffi.NULL, self._ffi.NULL, self._ffi.NULL
-        )
-        if x509 == self._ffi.NULL:
-            self._consume_errors()
-            raise ValueError(
-                "Unable to load certificate. See https://cryptography.io/en/"
-                "latest/faq.html#why-can-t-i-import-my-pem-file for more"
-                " details."
-            )
+    def load_pem_x509_certificate(self, data: bytes) -> x509.Certificate:
```

Comment:
Hmm, why change them at the backend layer vs. doing so at the x509 layer and removing these methods?